test/config/testini.ini

```diff
@@ -43,5 +43,10 @@ name = \
 
 [multi5]
 name = r.sub==p.sub \
-   &&r.obj==p.obj\
+   &&r.obj==p.obj \
    \
+
+[multi6]
```

Comment:
We need a new example like:

```ini
[matchers]
m = g(r.sub, p.sub, r.cid+'::'+r.lid) \
&& r.method == p.method \
&& keyMatch(r.cid, p.cid) \
&& keyMatch(r.lid, p.lid) \
&& keyMatch2(r.obj, p.obj) 
```

The key point here is that there should be one line that contains no "="

The `[multi6]` example doesn't seem to show such a corner case.

Reply:
In fact,the problem was caused by the empty line at the end of file,but not the `=` existed or not

Reply:
@Zxilly Oh, this is really a strange bug then. Nice catch.

@hariprasadiit is this the case?

Reply:
@hsluoyz yes, seems it's empty line issue. If we don't have empty line at the end, error is thrown.

@Zxilly Nice catch 👍